packages/subscription/src/stream.ts

```diff
@@ -56,6 +56,10 @@ export function createStream<T, TReturn = undefined>(callback: Callback<T, TRetu
 
     filter,
 
+    init: function*(task: Task) {
+      return subscribe(task);
+    },
```

Comment:
Well that's kinda cool!
```suggestion
    *init(task: Task) {
      return subscribe(task);
    },
```